include/spdlog/details/os.h

```diff
@@ -43,7 +43,7 @@
 #ifdef __linux__
 #include <sys/syscall.h> //Use gettid() syscall under linux to get thread id
 
-#elif __FreeBSD__
+#elif defined(__FreeBSD__) && !defined(__ORBIS__) // <sys/thr.h> is missing on Orbis/PS4
```

Comment:
Maybe remove the `ps4` and just leave orbis

Reply:
OK, will do